kong/plugins/request-transformer/handler.lua

```diff
@@ -1,15 +1,18 @@
+local BasePlugin = require "kong.plugins.base_plugin"
 local access = require "kong.plugins.request-transformer.access"
 
+local RequestTransformerHandler = BasePlugin:extend()
```

Comment:
Make it a simple empty table instead: `RequestTransformerHandler = {}`.

Alternatively, make it a table with the `PRIORITY` and `VERSION` fields set inside.